runtime/compiler/runtime/Listener.cpp

```diff
@@ -344,6 +344,15 @@ static int32_t J9THREAD_PROC listenerThreadProc(void * entryarg)
 
    j9thread_set_name(j9thread_self(), "JITServer Listener");
 
+   if (TR::Options::getVerboseOption(TR_VerboseJITServer))
```

Comment:
I would prefer this to be gated by  ` if (TR::Options::isAnyVerboseOptionSet())`, so even if we ask for compilation starts/end, we still get the information when the server started.